CHANGELOG.md

```diff
@@ -88,7 +91,7 @@ at anytime.
   * unnecessary `TempBlobManager` class
   * old storage classes used by the file manager, wallet, and blob manager
   * old `.db` database files from the data directory
-  *
+  * short(single dashed) arguments
```

Comment:
short (single dashed) arguments for `lbrynet-cli`

Reply:
Ok